manim/mobject/types/point_cloud_mobject.py

```diff
@@ -150,54 +170,58 @@ def thin_out(self, factor=5):
             )
         return self
 
-    def sort_points(self, function=lambda p: p[0]):
+    def sort_points(self, function: Callable[[Any], float] = lambda p: p[0]) -> Self:
```

Comment:
Does it need to be `Any`?
```suggestion
    def sort_points(self, function: Callable[[npt.NDArray[ManimFloat]], float] = lambda p: p[0]) -> Self:
```

Reply:
Good idea. Implemented.